README.md

```diff
@@ -1 +1,73 @@
-# judgement-day
\ No newline at end of file
+# judgement-day
+***
+## Feel Free to Rate FACN2 Mentors
+***
+### How to use our Website:
+To check out the app, Please clone this Repository.
+```javascript
+git clone  https://github.com/facn2/judgement-day.git
```

Comment:
@Karyum surely we can use whichever link we want? Either would work
git@github.com:facn2/judgement-day.git
https://github.com/facn2/judgement-day.git